manifests/kustomize/base/pipeline/ml-pipeline-persistenceagent-deployment.yaml

```diff
@@ -36,4 +36,15 @@ spec:
           requests:
             cpu: 120m
             memory: 500Mi
+        volumeMounts:
+          - mountPath: /var/run/secrets/kubeflow/tokens
+            name: persistenceagent-sa-token
       serviceAccountName: ml-pipeline-persistenceagent
+      volumes:
+        - name: persistenceagent-sa-token
+          projected:
+            sources:
+              - serviceAccountToken:
+                  path: persistenceagent-sa-token
+                  expirationSeconds: 3600
+                  audience: pipelines.kubeflow.org
```

Comment:
This is the place instructing Kubernetes to create a new token with a specific audience, expirity time & the path where the token to be loaded